### ISSUE TITLE
Support for 3270 screens

### ISSUE THREAD

**Issue by Harm10** [robocorp/rpaframework#143]
I know old screens but still widely used in big companies for legacy software and applications: 3270 screens.
I would like RPA to incorporate something for this maybe by using the great interface software x3270 (also available on GitHub: [)](https://github.com/pmattes/x3270)

**Comment by osrjv** [comment]
Thanks for the idea, I know terminals/mainframes are automated in some RPA processes. I'll link this to the library requests issue: https://github.com/robocorp/rpaframework/issues/87. However, I can't say _when_ this would be implemented.

Have you tried this existing library: https://github.com/Altran-PT-GDC/Robot-Framework-Mainframe-3270-Library? Any examples of use-cases or missing features in that library that come to mind would of course help our development later on.

**Comment by Harm10** [comment]
Yes I have tried the existing library and also am in contact with one of its developers. It is missing some functionality which will be incorporated as soon as the developer and/or me have time to pick it up.
But the charm of a framework is that it incorporates different features and functionality in 1 package which makes life a lot easier in Robot Framework land............. :-)

**Comment by tonnitommi** [comment]
Closing the issue for now. Can be re opened with detailed needs later on.
